src/General/Typography/Paragraph.tsx

```diff
@@ -15,6 +15,7 @@ const Paragraph: React.FunctionComponent<Props> = props => {
     bold,
     color = SecondaryColor.black,
     ellipsis,
+    lineHeight = false,
```

Comment:
1. it's hard to understand what this prop does, maybe name it `shouldSetLineHeight`?
2. why do we need this prop? when we use this component, why we want the line-height to be `normal`?

Reply:
The design has two types of paragraph, different for employers and candidates sites. One has line-height, for the other it is normal

Reply:
Updated